lib/suggestions-provider.js

```diff
@@ -10,7 +10,7 @@ export default class {
   constructor(packagesIndex) {
     this.packagesIndex = packagesIndex;
     // This will work on JavaScript and CoffeeScript files, but not in js comments.
-    this.selector = '.source.js, .source.ts, .source.tsx';
+    this.selector = '.source.js, .source.ts, .source.tsx, .source.cofee';
```

Comment:
Should be `.source.coffee`. Missing an "f".

Reply:
good catch, I didn't see the typo

Reply:
I always do that (I'm spoiled by autocomplete). I changed it in the github editor otherwise I would have caught it.